apps/condo/domains/billing/schema/BillingReceipt.test.js

```diff
@@ -578,13 +584,97 @@ describe('BillingReceipt', () => {
             expect(objs).toHaveLength(0)
         })
 
+        test('resident can read his own billingReceipts', async () => {
+
+            const { residentClient, receipt } = await makeResidentClientWithOwnReceipt()
+
+            const objs = await BillingReceipt.getAll(residentClient, {}, { raw:true })
```

Comment:
To cover a case, when there is no receipt records but his own, you need to generate BillingReceipts, belonging to another resident.